CopilotKit/packages/react-core/src/hooks/use-copilot-runtime-client.ts

```diff
@@ -115,20 +109,15 @@ export const useCopilotRuntimeClient = (options: CopilotRuntimeClientHookOptions
           // Process all errors as banners
           graphQLErrors.forEach(routeError);
         } else {
-          const isDev = shouldShowDevConsole(showDevConsole ?? false);
-          if (!isDev) {
-            console.error("CopilotKit Error (hidden in production):", error);
-          } else {
-            // Route non-GraphQL errors to banner as well
-            const fallbackError = new CopilotKitError({
-              message: error?.message || String(error),
-              code: CopilotKitErrorCode.UNKNOWN,
-            });
-            setBannerError(fallbackError);
-            // Trace the non-GraphQL error
-            traceUIError(fallbackError, error);
-            // TODO: if onError & renderError should work without key, insert here
-          }
+          // Route non-GraphQL errors to banner as well
+          const fallbackError = new CopilotKitError({
+            message: error?.message || String(error),
+            code: CopilotKitErrorCode.UNKNOWN,
+          });
+          setBannerError(fallbackError);
+          // Trace the non-GraphQL error
+          traceUIError(fallbackError, error);
```

Comment:
Does this mean that the ErrorBanner renders when `showDevConsole` is set to false as well? If so, I think what we really want is this:

```suggestion
          if(isDev) setBannerError(fallbackError);
          // Trace the non-GraphQL error
          traceUIError(fallbackError, error);
```

Do I understand this correctly?

Reply:
No, the banner is not rendered. It is set, and somewhere else, if this is dev mode, it will be rendered.